### ISSUE TITLE
LF 0.20.3+ error with IE8

### ISSUE THREAD

**Issue by cbeckfield** [ef4/liquid-fire#311]
Having issues with 0.20.3 and 0.20.4 on IE8. Using Ember 1.13.2. On page load it gives this error:

    SCRIPT1010: Expected identifier

Which then points to line 7 in liquid-measured.js:

    exports.default = liquid_measured.default;

Stepping back to 0.20.2 fixes the issue.

**Comment by rwjblue** [comment]
Liquid Fire is almost certainly not going to work properly under IE8 (I do not have proof), but I would not expect to get a parse error either.

---

It probably surfaced due to the `export { default } from "liquid-fire/components/liquid-measured";` line in `liquid-measured.js`, my guess is that esperanto does not honor the `_evilES3SafeReExports` flag in that case.

You can attempt to fix via:

```javascript
import Thing from 'liquid-fire/components/liquid-measured';

export default Thing;
```

If that fixes the syntax error for IE8, you should consider submitting a PR so @ef4 can decide if he wants to support IE8..

**Comment by xcskier56** [comment]
I have run into this issue as well, but in the interviening time it seems that the `export { default } from "liquid-fire/components/liquid-measured"` has been expanded to export the `measure` function.

 `export { default, measure } from "liquid-fire/components/liquid-measured"` 

@rwjblue The fix that you proposed above seemed to work for other locations that have this issue, but I'm not familiar enough with the ES6 syntax to figure out how to do the double import/export. If this can be done I'll happily make a PR for it.

**Comment by rwjblue** [comment]
```
import DefaultThing, { measure } from 'liquid-fire/components/liquid-measured';

export default DefaultThing;
export measure;
```

**Comment by stefanpenner** [comment]
i dont think we can reasonable go around fixing all packages, lets wait for esperanto to fix itself.

For users affected, they can run into: https://github.com/stefanpenner/es3-safe-recast

**Comment by xcskier56** [comment]
Sounds good to me. Thanks for the help.

**Comment by stefanpenner** [comment]
looks like ember-cli master has the latest esperanto, it likely works as expected.
